src/app/timer/timer.tsx

```diff
@@ -47,7 +48,8 @@ export const Timer: React.FC = () => {
 
         return (
             <Flex justify="center" align="center" mt="xl" direction="column">
-                <h1>Sesion invalid, {invalidReason}</h1>
+                <AccessDeniedAlert />
+                <br />
                 <Button onClick={() => onSignInClick(clerk)}>Sign in</Button>
```

Comment:
this is fine, but `<br/>` tags are pretty obsolete at this point.  It's preferred to wrap in a flexbox container, like Flex or Stack.  Doing so lets you set the `gap` and adjust alignment if needed.